docs/uplink/rest-api.md

```diff
@@ -6,19 +6,39 @@ A quick note on TLS: when you expose the uplink API over the Internet, you shoul
 
 ## Installation
 
-The Uplink Client API can be enabled through the helm chart. Update the `values.yaml`:
+The Uplink Client API can be enabled through the helm chart.
```

Comment:
The REST API is part of the client-api component, and is disabled by default.

You'll need to configure authentication, then update the values.yaml file and install the chart again.